dev-tools/packer/platforms/centos/run.sh.j2

```diff
@@ -21,26 +21,30 @@ fi
 RPM_VERSION=`echo ${VERSION} | sed 's/-/_/g'`
 
 # create rpm
-fpm --force -s dir -t rpm \
-        -n {{.beat_pkg_name}} -v ${RPM_VERSION} \
-        --architecture {{.rpm_arch}} \
-        --vendor "{{.beat_vendor}}"  \
-        --license "{{.beat_license}}"  \
-        --description "{{.beat_description}}" \
-        --url {{.beat_url}} \
-        --rpm-init /tmp/{{.beat_pkg_name}}.init \
-        --after-install /tmp/systemd-daemon-reload.sh \
-        --config-files /etc/{{.beat_name}}/{{.beat_name}}.yml \
-        homedir/=/usr/share/{{.beat_name}} \
-        beatname-${RUNID}.sh=/usr/bin/{{.beat_name}}.sh \
-        {{.beat_name}}-linux-{{.arch}}=/usr/share/{{.beat_name}}/bin/{{.beat_name}} \
-        {{.beat_name}}-linux.yml=/etc/{{.beat_name}}/{{.beat_name}}.yml \
-        {{.beat_name}}-linux.reference.yml=/etc/{{.beat_name}}/{{.beat_name}}.reference.yml \
-        fields.yml=/etc/{{.beat_name}}/fields.yml \
-        ${RUNID}.service=/lib/systemd/system/{{.beat_pkg_name}}.service \
-        god-linux-{{.arch}}=/usr/share/{{.beat_name}}/bin/{{.beat_name}}-god
-
+COMMAND='fpm --force -s dir -t rpm
```

Comment:
Maybe instead of building up `COMMAND`, we could build up `ARGS`, and then we run `fpm $ARGS`? It's the same thing, but I think it's that's more common as a pattern.